cli/utils/init.config.ts

```diff
@@ -13,34 +14,37 @@ export default function provideInitConfig(
   filesystem: typeof fs,
   fortaKeystore: string,
   configFilename: string,
-  agentId: string,
-  getFortaConfig: GetFortaConfig
+  contextPath: string,
+  localConfigFilename: string,
```

Comment:
localConfigFilename is typically used for the `run` command with the `config` param (e.g. your bot scans multiple networks and you have a config file for each network so you dont need to keep changing jsonRpcUrl, instead you just specify the file using `config` param). however, when we are running the `init` command we should ignore this and always write to a file called forta.config.json since we are initializing a brand new project

Reply:
Using the `configFilename` instead of `localConfigFilename`